### ISSUE TITLE
[HttpClient] CURLOPT_STDERR resource has gone away

### ISSUE THREAD

**Issue by versh23** [symfony/symfony#31803]
**Symfony version(s) affected**: 4.3

**Description**  
I try to send POST request 

```
$httpClient = HttpClient::create();
            $response = $httpClient->request('POST', 'http://ttt:8080/', [
                'json' => [
                    'variables' => [
                    ],
                ]
            ]);
```

and get

`app_1         | [03-Jun-2019 09:01:28] WARNING: [pool www] child 9 said into stderr: "NOTICE: PHP message: PHP Fatal error:  Uncaught ErrorException: Warning: curl_multi_exec(): CURLOPT_STDERR resource has gone away, resetting to stderr in /app/vendor/symfony/http-client/Response/CurlResponse.php:259"
`

**Additional context**  
it is fresh sf4.3 with docker image php:7.3-fpm-alpine

Im try to write in php://temp directly and read - its work, so stream is OK

If i comment line in vendor/symfony/http-client/Response/CurlResponse.php::44 - its work

**Comment by nicolas-grekas** [comment]
Can you try commenting out L170&172 in CurlResponse.php, does it fix the issue?
https://github.com/symfony/symfony/blob/693cbff64766d3a2b8c143757b8340a0c8330c2a/src/Symfony/Component/HttpClient/Response/CurlResponse.php#L170-L171

**Comment by versh23** [comment]
@nicolas-grekas yes, it works

also, i work in docker on windows , if it matters

**Comment by nicolas-grekas** [comment]
Thanks, should be fixed by #31832, please report if not
